content/docs/command-reference/metrics/add.md

```diff
@@ -18,42 +18,36 @@ defines the given `path` as an <abbr>output</abbr>, marking `path` as a metric
 file to track.
 
 Note that outputs can also be marked as metrics via the `-m` or `-M` options of
-`dvc run`.
+`dvc run`. We recommend using `-M` option to keep metrics in Git history.
```

Comment:
How about not even mentioning `-m` anymore? It's getting removed according to https://github.com/iterative/dvc/issues/3409#issuecomment-594130496 anyway.

Reply:
Let's keep it for now. We can remove it later.

Reply:
Kk